composer.json

```diff
@@ -49,6 +48,10 @@
         "test:unit": [
             "[ -e .Build/bin/phpunit ] || composer update",
             "TYPO3_PATH_ROOT=.Build/Web .Build/bin/phpunit -c .Build/vendor/nimut/testing-framework/res/Configuration/UnitTests.xml Tests/Unit/ --debug --verbose"
+        ],
+        "test:functional": [
+            "[ -e .Build/bin/phpunit ] || composer update",
+            "TYPO3_PATH_ROOT=$PWD/.Build/Web typo3DatabaseName=\"test\" typo3DatabaseHost=\"127.0.0.1\" typo3DatabaseUsername=\"root\" typo3DatabasePassword=\"root\" .Build/bin/phpunit -c Tests/Functional/FunctionalTests.xml Tests/Functional"
```

Comment:
You can use single quotes, then you don't need to escape them.

Reply:
right :)